### DISCUSSION TITLE
Http to https

### DISCUSSION THREAD

**Discussion by Michelle-Luttringer** [GetPublii/Publii#1379] [Help]
I startet my website on http. Now the ssl certificate got installed. So I went to server configuration and swiched to https. But when I sync my website it's still on http. Is there anything else I have to do so that it works. Or how do I check where the problem is?

**Comment by dziudek** [comment]
Please check if website works with https url - if yes, then you must add https redirects under your server configuration

**Reply by Michelle-Luttringer** [reply]
How do I check that?

**Reply by dziudek** [reply]
Just write your website URL with https:// at the beginning :) If it will load properly then your issue is with lack of https redirect

**Reply by Michelle-Luttringer** [reply]
No that does not work.

**Reply by dziudek** [reply]
Please provide an URL to your website

**Reply by Michelle-Luttringer** [reply]
fantasy.luttringer.ch

**Reply by dziudek** [reply]
@Michelle-Luttringer - your website displays correctly under https://fantasy.luttringer.ch/ but your certificate is probably not installed. Please contact your hosting and:

1) Ask for install your SSL certificate
2) Ask for enabling force redirect to HTTPS the traffic on your website

**Reply by Michelle-Luttringer** [reply]
The admin of Luttringer.ch installed it on the main domain. And I checked the main domain, and it works. So I guess the problem is, that I am using a subdomain. Is there a way, that the subdomain can use the certificate from the main domain?

**Reply by dziudek** [reply]
Yes, you need a wildcard ssl certificate to use SSL on subdomains. But this topic has nothing to do with Publii - I have described what you need to do - please continue this case with your admins :)

**Reply by Michelle-Luttringer** [reply]
Ok, thanks for your help
